src/fission.ts

```diff
@@ -40,11 +27,29 @@ export default class Fission {
   url(cid: CID): string {
     return `${this.baseURL}/ipfs/${cid}`
   }
+}
 
-  async add(content: Content, name?: string): Promise<CID> {
-    if (!this.auth) {
-      throw new Error('Must be logged in to add content to IPFS')
+export class FissionUser extends Fission {
+  auth: {
+    username: string
+    password: string
+  }
+
+  constructor(baseURL: string, username: string, password: string) {
```

Comment:
Actually, come to think if it, some folks may want to skip the `new Fission` step entirely. We could do `constructor(username: string, password: string, baseURL?: string)`. Thoughts?